lib/puppet/reports/tagmail.rb

```diff
@@ -201,44 +210,44 @@ def send(reports)
 
     # Starting a new thread has been commented out as it causes conflict with IO.popen, where the thread just dies
     # after the first run.
-    #Thread.new {
-      if tagmail_conf[:smtpserver] and tagmail_conf[:smtpserver] != "none"
-        begin
-          Net::SMTP.start(tagmail_conf[:smtpserver], tagmail_conf[:smtpport], tagmail_conf[:smtphelo]) do |smtp|
-            reports.each do |emails, messages|
-              smtp.open_message_stream(tagmail_conf[:reportfrom], *emails) do |p|
-                p.puts "From: #{tagmail_conf[:reportfrom]}"
-                p.puts "Subject: Puppet Report for #{self.host}"
-                p.puts "To: " + emails.join(", ")
-                p.puts "Date: #{Time.now.rfc2822}"
-                p.puts
-                p.puts messages
-              end
-            end
-          end
-        rescue => detail
-          message = "Could not send report emails through smtp: #{detail}"
-          Puppet.log_exception(detail, message)
-          raise Puppet::Error, message, detail.backtrace
-        end
-      else
-        begin
+    # Thread.new {
```

Comment:
Below seems to have changes substantially. Assuming the unit tests and acceptance tests work. I am ok with it.

Reply:
Looks like code was still indented for commented-out `Thread.new {` and reviewing with `?w=1` makes the change much more clear.